### PR TITLE
[main<] Telemetry

### PR DESCRIPTION
**Pull request by vpavicic**
### Description

Need to update telemetry based on a PR from 2.12.
https://github.com/memgraph/memgraph/pull/1337

### Pull request type

Please check what kind of PR this is:

- [ ] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
(especially necessary if the PR is related to a release)

Closes:
(paste the link to the issue it closes)

### Checklist:

- [ ] Check all content with Grammarly
- [ ] Perform a self-review of my code
- [ ] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors
- [ ] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

File: pages/fundamentals/telemetry.mdx [hunk 1]
```diff
@@ -23,19 +23,33 @@ While a Memgraph database instance is running and an open internet connection is
...
+  - client name
+  - supported and agreed Bolt version
+  - number of sessions from each clients
+  - number of executed queries
+  - authentication type (user or anonymous)
+
+- **Information about active databases**
+  - disk storage and usage
+  - RAM usage
+  - if durability and SSL is enabled
```
Comment (gitbuda) [inline pages/fundamentals/telemetry.mdx:42]: SSL is on the Memgraph instance level (not under each database config)? @MarkoBarisic please verify 🙏

File: pages/fundamentals/telemetry.mdx [hunk 2]
```diff
@@ -23,19 +23,33 @@ While a Memgraph database instance is running and an open internet connection is
...
+  - number of nodes and relationships
+  - number of constraints and indices
+  - storage mode  
 
 - **Database runtime information**
-    - CPU usage
-    - Memory usage
-    - The number of nodes and relationships stored in the database
+    - Timestamp of the first failed and first successful exwcuted query
```
Comment (gitbuda) [inline pages/fundamentals/telemetry.mdx:48]: typo, `executed`

File: pages/fundamentals/telemetry.mdx [hunk 3]
```diff
@@ -23,19 +23,33 @@ While a Memgraph database instance is running and an open internet connection is
...
+  - Host OS
+  - Kernel information
+<br/>    
+
+- **Information about the clients**
+  - client name
+  - supported and agreed Bolt version
+  - number of sessions from each clients
+  - number of executed queries
+  - authentication type (user or anonymous)
```
Comment (MarkoBarisic) [inline pages/fundamentals/telemetry.mdx:37]: This is called connection type in our telemetry database, @gitbuda is authentication type ok?

File: pages/fundamentals/telemetry.mdx (hunk 1 above)
Comment (MarkoBarisic) [reply pages/fundamentals/telemetry.mdx:42]: yes SSL should be on Memgraph instance level

Comment (MarkoBarisic) [reply pages/fundamentals/telemetry.mdx:42]: so leave it to be just `if durabilty and SSL is enabled`

File: pages/fundamentals/telemetry.mdx [hunk 4]
```diff
@@ -23,19 +23,33 @@ While a Memgraph database instance is running and an open internet connection is
...
+
+- **Information about active databases**
```
Comment (MarkoBarisic) [inline pages/fundamentals/telemetry.mdx:39]: Add isolation level (example is snapshot isolation)

**Pull request merged**